pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -42,23 +42,25 @@ const (
 	trustedCABundleConfigMap = "config-trusted-cabundle"
 	clusterInterceptors      = "openshift-pipelines-clusterinterceptors"
 	namespaceVersionLabel    = "openshift-pipelines.tekton.dev/namespace-reconcile-version"
+	disableRbacInjection     = "openshift-pipelines.tekton.dev/disable-rbac-injection"
 )
 
 // Namespace Regex to ignore the namespace for creating rbac resources.
 var nsRegex = regexp.MustCompile(common.NamespaceIgnorePattern)
 
 type rbac struct {
-	kubeClientSet     kubernetes.Interface
-	operatorClientSet clientset.Interface
-	manifest          mf.Manifest
-	ownerRef          metav1.OwnerReference
-	version           string
+	kubeClientSet          kubernetes.Interface
+	operatorClientSet      clientset.Interface
+	manifest               mf.Manifest
+	ownerRef               metav1.OwnerReference
+	version                string
+	disableRbacClusterWide bool
 }
 
 func (r *rbac) cleanUp(ctx context.Context) error {
```

Comment:
should we add cleanup for injection label too? like too remove the label when uninstalling from ns

Reply:
Not sure whether we can clean the label as part of Operator uninstall as label is injected by user and not by Operator so i am thinking let user remove it based on their use case
But i may be wrong in considering end user scenario

/cc @ppitonak @vdemeester

Reply:
ah sry I meant while deleting `TektonConfig` 😅 .. while uninstalling it would be kinda difficult to do this

Reply:
even while deleting `TektonConfig` it won't do anything with labels on namespace because labels are set by User

Reply:
> so i am thinking let user remove it based on their use case

i think, this will be the expected UX from a user side.
as this label is not added by the controller, let us not touch it from the controller.

usecase: if a user wants to uninstall reinstall, the user (admin) will not want to relabel all the namespaces, again. The user might expect the re-installation to come back to the state based on already existing user supplied labels.